frontend/packages/core/src/Feedback/stories/warning.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+// TODO: remove when https://github.com/lyft/clutch/pull/607/files#r512255592 lands
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { action } from "@storybook/addon-actions";
+import type { Meta } from "@storybook/react";
+
+import type { WarningProps } from "../warning";
+import Warning from "../warning";
+
+export default {
+  title: "Core/Warning",
```

Comment:
should we move it to  "Core/Feedback/Warning" since it's now part of the feedback folder?